internal/person.go

```diff
@@ -0,0 +1,21 @@
+package internal
+
+type Person struct {
+	CompareValue   string
+	ID             string
+	Attributes     map[string]string
+	DisableChanges bool
+}
+
+func (p *Person) Matches(filters Filters) bool {
+	for _, f := range filters {
+		value := p.Attributes[f.Attribute]
```

Comment:
What if the f.Attribute doesn't match any of the p.Attributes **keys**?

Should this be something like
```
value, ok := p.Attributes[f.Attribute]
if !ok {
   continue
}
...
```

Reply:
Not necessarily. I wrote the documentation to cover this. If the attribute value is missing or empty, the record is omitted. So `false` is the correct return value in this case. Are you thinking this treatment is problematic?

Reply:
What if f.Attribute has a typo? Won't that mean that no records will be included?

Reply:
Yes, I will cause the sync to fail if the attribute is missing.